### PR TITLE
[MM-37377] - LHS does not update after Getting Started steps are complete

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
- Makes sure that when Getting Started steps are finished, the sidebar updates to "Next Steps and Tips"
- Fixes Modal text when closing Getting Started and Next Steps and Tips

#### Ticket Link
[MM-37377](https://mattermost.atlassian.net/browse/MM-37377?atlOrigin=eyJpIjoiNzA4Y2VjNTQwODFiNGExMzkwYTRjMGJmNjRkNDI5NjkiLCJwIjoiaiJ9)

#### Release Note
```release-note
NONE

```

### CODE REVIEW

**Review by pablovelezvidal — APPROVED** [review]
LGTM! 🚢 , just one comment, I think the only translation file we modify is the en.json, maybe @marianunez can confirm that!

**Review by marianunez — CHANGES_REQUESTED** [review]
Can we cover this case with a cypress test as well?

File: i18n/en_AU.json [hunk 1]
```diff
@@ -3937,7 +3937,7 @@
   "remove_next_steps_modal.confirm": "Remove",
   "remove_next_steps_modal.header": "Remove {title}?",
   "remove_next_steps_modal.helpText": "Access {title} any time through the Main Menu",
-  "remove_next_steps_modal.mainText": "This will remove this section from your sidebar, but you can access it later in the Tips and Next steps section of the Main Menu.",
+  "remove_next_steps_modal.mainText": "This will remove this section from your sidebar, but you can access it later in the {title} section of the Main Menu.",
```
Comment (marianunez) [inline i18n/en_AU.json:3940]: That is correct, we should not update other translations files than `en.json`. The updates in the main file will be reflected in our translation servers and the specific languages will show up to be changed as well by translators.

**Review by marianunez — APPROVED** [review]
LGTM! Thanks @AGMETEOR!

**Comment by AGMETEOR** [comment]
Hello @stevemudie QA steps are on the ticket for this one.

**Review by stevemudie — COMMENTED** [review]
@AGMETEOR Unfortunately, the tests fail.

**Getting Started** persists in the LHS at every point of the QA steps you described in the ticket.

**Comment by AGMETEOR** [comment]
@stevemudie can't seem to reproduce locally. Maybe something is up with the spinwicks. I'll ask someone else to test locally as well to verify that the issue maybe with the spinwicks.

**Comment by AGMETEOR** [comment]
@stevemudie we have confirmed that indeed the spinwick is not using the code for this PR. Maybe I'll ask someone from SRE to help out. 

As you can see the debugger below, the code deployed by the spinwick is different
-----
Screenshot 2021-07-29 at 22 33 51

Thanks to @nickmisasi for helping to confirm

**Comment by AGMETEOR** [comment]
> Mattermost test server created! 🎉
> 
> Access here: https://mattermost-webapp-pr-8478.test.mattermost.cloud
> 
> Account Type	Username	Password
> Admin	sysadmin	Sys@dmin123
> User	user-1	User-1@123

Hello @stevemudie the spinwick issue has been resolved correctly.

**Review by stevemudie — APPROVED** [review]
Tested and passed.

**Getting Started** changed to **Tips & Next Steps** as expected once the GS was complete including when dismissing the T&NS.

**Pull request merged**